tests/sections/ui/tooltip/Tooltip.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { fireEvent, render } from '@testing-library/react'
+import { Tooltip, TooltipProps } from '../../../../src/sections/ui/tooltip/Tooltip'
+
+describe('Tooltip', () => {
+  const defaultProps: TooltipProps = {
+    placement: 'top',
+    message: 'This is a tooltip message'
+  }
+
+  it('renders without crashing', () => {
+    render(<Tooltip {...defaultProps} />)
+  })
+  it('renders the tooltip on mouseOver', async () => {
+    const { container, findByRole } = render(<Tooltip {...defaultProps} />)
+    const svg = container.querySelector('span > svg') as HTMLElement
+    expect(svg).toBeInTheDocument()
```

Comment:
We'll try to see if we can get the svg with some getBy* function and if not we can go this approach

Reply:
We added an img role to the span, so we can use getByRole in the test.